doc/source/f2py/f2py.getting-started.rst

```diff
@@ -233,26 +232,25 @@ suitable for wrapping (e.g. third party) Fortran codes for which
 modifications to their source codes are not desirable nor even
 possible.
 
-However, if editing Fortran codes is acceptable, then the generation
-of an intermediate signature file can be skipped in most
-cases. Namely, F2PY specific attributes can be inserted directly to
-Fortran source codes using the so-called F2PY directive. A F2PY
-directive defines special comment lines (starting with ``Cf2py``, for
-example) which are ignored by Fortran compilers but F2PY interprets
-them as normal lines.
+However, if editing Fortran codes is acceptable, then the generation of an
+intermediate signature file can be skipped in most cases. Namely, F2PY specific
+attributes can be inserted directly to Fortran source codes using the so-called
```

Comment:
"directly into Fortran"

Reply:
Omit "so-called"

Reply:
Replacing this with:

> Namely, F2PY specific
attributes can be inserted directly into Fortran source codes using  F2PY
directives.